Sources/Extensions/CoreAnimation/CALayerExtensions.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  CALayerExtension.swift
+//  SwifterSwift
+//
+//  Created by Morgan Dock on 4/26/18.
+//  Copyright © 2018 SwifterSwift
+//
+
+public extension CALayer {
+
+    /// SwifterSwift: Create a shadow affect, using the same parameters used in design software such as Sketch, Zeplin, and others.
+    /// This Extension originates from here: https://stackoverflow.com/questions/34269399/how-to-control-shadow-spread-and-blur/34270362
+    ///
+    /// - Parameters:
+    ///   - color: the tint of the shadow effect.  Default is UIColor.black
+    ///   - alpha: opacity of the color applied to the shaddow effect.  Default is 0.5
+    ///   - x: the horizontal offset applied to the shadow effect.  Default is 0.
+    ///   - y: the vertical offset applied to the shadow effect.  Default is 2.
+    ///   - blur: if set to 0 the shadow will be sharp, the higher the number, the more blurred it will be.  Default is 4.
+    ///   - spread: positive values increase the size of the shadow, negative values decrease the size. Default is 0 (the shadow is same size as blur).
+    public func applyShadow(color: UIColor = .black, alpha: Float = 0.5, xOffset: CGFloat = 0, yOffset: CGFloat = 2, blur: CGFloat = 4, spread: CGFloat = 0) {
```

Comment:
These default values seem a bit random. Are they simply what you use for your project? I think it would make more sense to use the default values for their mapped values (e.g. `(0.0, -3.0)` for `xOffset` and `yOffset` respectively, `3.0` for `blur`).